projects/kubernetes-sigs/image-builder/packer/ami/additional_files/generate-kube-vip-manifest.sh

```diff
@@ -20,8 +19,6 @@ spec:
     env:
     - name: vip_arp
       value: "true"
-    - name: vip_interface
-      value: $DNI
```

Comment:
Newer version of kube-vip auto-detects the right interface to use. Let me know if you still want me to keep this for some other reason @g-gaston